x-pack/legacy/plugins/integrations_manager/common/constants.ts

```diff
@@ -9,12 +9,12 @@ import manifest from '../kibana.json';
 
 export const PLUGIN = {
   ID: manifest.id,
-  TITLE: i18n.translate('xpack.integrationsManager.pluginTitle', {
-    defaultMessage: 'Integrations Manager',
+  TITLE: i18n.translate('xpack.epm.pluginTitle', {
+    defaultMessage: 'Elastic Package Manager',
   }),
-  DESCRIPTION: 'Install and manage your elastic data ingest integrations',
+  DESCRIPTION: 'Install and manage your Elastic data ingest packages',
```

Comment:
Not really related to this change -- are we dealing with `ingest packages` or just `packages`?

Reply:
I'm not sure, tbh. All this copy still need reviews & updates before launch.